drivers/core_demographics_report/app/models/core_demographics_report/disability_calculations.rb

```diff
@@ -13,6 +13,7 @@ def disability_detail_hash
         HudUtility2024.disability_types.each do |key, title|
           hashes["disability_#{key}"] = {
             title: "Disability #{title}",
+            cannot_view_details: ! can_view_client_disability?(@filter.user, key),
```

Comment:
might be easier to read if this was:
```suggestion
            can_view_details:  can_view_client_disability?(@filter.user, key),
```
You'd have to invert the usage of it as well, but I find reasoning about the positive generally easier.

Reply:
This is updated in the latest commit